src/pathOr.js

```diff
@@ -0,0 +1,30 @@
+var _curry3 = require('./internal/_curry3');
+
+
+/**
+ * If the given, non-null object has an own property at the given path, returns
+ * the value at that path. Otherwise returns the provided default value.
+ *
+ * @func
+ * @memberOf R
+ * @category Object
+ * @sig a -> [String] -> {k: v} -> v | a
```

Comment:
I suggest `a -> [String] -> Object -> a` as the type signature. The default value should be of the same type as the expected value.